q2_mlab/tests/test_unit_benchmark.py

```diff
@@ -188,7 +189,54 @@ def testReturnBestModel(self):
         )
 
         self.assertTrue(best_model)
-        self.assertEquals(results['ACCURACY'].max(), best_accuracy)
+        self.assertEqual(results['ACCURACY'].max(), best_accuracy)
+
+    def testResultTable(self):
+
+        n_features = 10
+        n_samples = 30
+        test_data = np.arange(n_features*n_samples).reshape(
+            n_features, n_samples
+        )
+        sample_ids = [f"S{i}" for i in range(n_samples)]
+        obs_ids = [f"F{i}" for i in range(n_features)]
+        test_table = biom.Table(test_data, obs_ids, sample_ids)
+
+        continuous_target = pd.Series(
+            [i for i in range(n_samples)],
+            index=sample_ids
+        )
+        discrete_target = pd.Series(
+            np.tile([0, 1], int(n_samples/2)),
+            index=sample_ids
+        )
+
+        # Test Regression table
+        results, _, _ = q2_mlab._unit_benchmark(
+            table=test_table,
+            metadata=continuous_target,
+            algorithm="LinearSVR",
+            params=self.svr_params,
+            n_repeats=1
+        )
+        for sampleid in results.SAMPLE_ID:
```

Comment:
I would recommend using Pandas' utilities for checking that two dataframes are almost equal:

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.testing.assert_frame_equal.html

Reply:
Agreed.